sympy/stats/drv_types.py

```diff
@@ -247,6 +236,17 @@ def Hermite(name, a1, a2):
     >>> variance(H)
     21
 
+    Parameters
+    ==========
+
+    a1 : A Positive number greater than equal to 0.
+    a2 : A Positive number greater than equal to 0.
```

Comment:
My reading of the style guide is that there should be no capitalisation and also the  indefinite article "a" is unnecessary so maybe this should be:
```
    a2 : positive number greater than equal to 0.
```
Also if this is "greater than *or* equal" then that would normally be described as "nonnegative" rather than "positive". 

I don't know if it actually makes sense for either parameter to be equal to zero...